models/demos/wormhole/llama31_8b/demo/demo.py

```diff
@@ -67,9 +67,9 @@ def preprocess_inputs(input_prompts, tokenizer, model_args, dtype, embd, instruc
 def run_llama_demo(user_input, batch_size, device, instruct_mode, is_ci_env):
     # Set Llama flags for CI
     if is_ci_env and instruct_mode:  # Update paths for instruct mode, otherwise use default paths for general weights
-        os.environ["LLAMA_CKPT_DIR"] = "/mnt/MLPerf/tt_dnn-models/Llama/Meta-Llama-3.1-8B-Instruct/"
-        os.environ["LLAMA_TOKENIZER_PATH"] = "/mnt/MLPerf/tt_dnn-models/Llama/Meta-Llama-3.1-8B-Instruct/"
-        os.environ["LLAMA_CACHE_PATH"] = "/mnt/MLPerf/tt_dnn-models/Llama/Meta-Llama-3.1-8B-Instruct/"
+        os.environ["LLAMA_CKPT_DIR"] = "/mnt/MLPerf/tt_dnn-models/llama/Meta-Llama-3.1-8B-Instruct/"
```

Comment:
what's the policy on setting env variables inside the tests? Is not it better to set them where the test is called?

Reply:
Infra wants to avoid setting any flags on test script.

For this specific code snippet, we set the default Llama directories to the pretrained weights in `model_config.py`. For the demo, since Instruct weights have "better outputs" (question<->answer) we update the location inside the demo script like this.